### PR TITLE
[TACO] Fix PasswordInput size prop

### PR DESCRIPTION
**Pull request by doug2k1**
[no-jira]

### Description

The `PasswordInput` component accepts the `size` prop, but it was not working properly. The component rendered in default size, regardless of the `size` value.

### How to test

- [Temploy](https://picasso.toptal.net/taco-fix-passwordinput-size)
- Check the new **Sizes** example in `PasswordInput` storybook

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| image | image |

### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- [x] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [x] Make sure that additions and changes on the design follow [Toptal's BASE design](https://design.toptal.net/), and it's been already discussed with designers at #-base-core
- [x] Annotate all `props` in component with documentation
- [x] Create `examples` for component
- [x] Ensure that deployed demo has expected results and good examples
- [x] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [x] Self reviewed
- [x] Covered with tests ([visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md) included)

**Breaking change**

- [-] codemod is created and showcased in the changeset
- [-] test alpha package of Picasso in StaffPortal

> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

### CODE REVIEW

File: .changeset/beige-clocks-float.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+'@toptal/picasso': patch
+'@toptal/picasso-forms': patch
```
Comment (sashuk) [inline .changeset/beige-clocks-float.md:3]: Can you please specify why `@toptal/picasso-forms` is bumped (`@toptal/picasso` is a peer dependency)?

```json
// packages/picasso-forms/package.json
  ..
  "peerDependencies": {
    "@toptal/picasso": "^42.0.0",
     ...
  },
```

Comment (doug2k1) [reply .changeset/beige-clocks-float.md:3]: I was unsure if that's needed. I'll remove it from the changeset. Thanks for pointing out it's a peer dependency.

Comment (sashuk) [reply .changeset/beige-clocks-float.md]: @doug2k1 Thank you!

**Review by sashuk — APPROVED** [review]
Tested on temploy, works as expected! Code looks good, I will wait for Happo checks to fail so I can approve new visuals for `PasswordInput`

**Comment by sashuk** [comment]
There is also an error in Lint check, it is not related to this pull request and will be fixed in https://github.com/toptal/picasso/pull/4152

**Comment by sashuk** [comment]
@toptal-anvil ping reviewers

**Review by TomasSlama — APPROVED** [review]
Thanks for the fix!

**Comment by sashuk** [comment]
@doug2k1 Can you please rebase pull request?

**Comment by doug2k1** [comment]
@sashuk I had to rebase the PR again and now Happo is failing on unrelated screens. Could you take a look? 🙏

**Pull request merged**